_data/sites.json

```diff
@@ -6086,6 +6096,17 @@
         ]
     },
 
+    {
+        "name": "Money Saving Expert",
+        "url": "https://clubs.moneysavingexpert.com/cheapenergyclub/my-account",
+        "difficulty": "easy",
+        "notes": "Just login to your account, select 'My Account', scroll to the bottom, select 'Settings' and click 'Close Account'",
+        "email": "energyclub@moneysavingexpert.com",
```

Comment:
```suggestion
```
If sending an email is not needed, I'd rather not list it here to avoid confusion